AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt

```diff
@@ -2344,6 +2346,15 @@ open class DeckPicker :
         }
     }
 
+    /** Disables the shortcut of the deck whose id is [did] */
+    fun disableDeckShortcut(did: DeckId) {
+        val decks = col.decks.childDids(did, col.decks.childMap()).toMutableList()
+        val deckString = decks.joinToString(",")
+        val deckIds = deckString.split(",").map { it.trim() }
+        val errorMessage: CharSequence = getString(R.string.deck_shortcut_doesnt_exist)
+        ShortcutManagerCompat.disableShortcuts(this, deckIds, errorMessage)
+    }
+
```

Comment:
This currently include only the deck children, but not the deck itself. Here's my suggestion

```suggestion
    /** Disables the shortcut of the deck whose id is [did] and its children */
    fun disableDeckAndChildrenShortcuts(did: DeckId) {
        val childDids = col.decks.childDids(did, col.decks.childMap()).map { it.toString() }
        val deckTreeDids = listOf(did.toString(), *childDids.toTypedArray())
        val errorMessage = getString(R.string.deck_shortcut_doesnt_exist)
        ShortcutManagerCompat.disableShortcuts(this, deckTreeDids, errorMessage)
    }
```